README.md

```diff
@@ -63,7 +64,7 @@ You don’t have to ever use `eject`. The curated feature set is suitable for sm
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/sinansevgi/mycalculator/issues/).
 
 ## Show your support
 
```

Comment:
- [ ] I noticed a mixture of `CSS `files with components, kindly create a folder to store your `CSS` files and import them where required.